packages/grafana-data/src/panel/PanelPlugin.ts

```diff
@@ -90,9 +122,11 @@ export class PanelPlugin<TOptions = any, TFieldConfigOptions = any> extends Graf
 
   /**
    * Enables configuration of panel's default field config
+   *
+   * @deprecated setFieldConfigDefaults is deprecated in favor of setCustomFieldOptions
    */
-  setFieldConfigDefaults(defaultConfig: Partial<FieldConfigSource>) {
-    this.fieldConfigDefaults = {
+  setFieldConfigDefaults(defaultConfig: Partial<FieldConfigSource<TFieldConfigOptions>>) {
+    this._fieldConfigDefaults = {
```

Comment:
I think we can remove this, is any plugin using it? was added like 1-2 weeks ago :)

Reply:
heh, tru dat, think we can remove this